test/jdk/javax/swing/JTableHeader/JTableHeaderLabelRightAlignTest.java

```diff
@@ -46,10 +46,16 @@ public class JTableHeaderLabelRightAlignTest {
     private static final int WIDTH = 300;
     private static final int HEIGHT = 150;
     private static final double SCALE = 2.25;
+    private static boolean testStatus = true;
+    private static String failureMsg;
 
     public static void main(String[] args) throws Exception {
         UIManager.setLookAndFeel("javax.swing.plaf.metal.MetalLookAndFeel");
         SwingUtilities.invokeAndWait(JTableHeaderLabelRightAlignTest::test);
+        if (!testStatus) {
+            throw new RuntimeException(failureMsg);
+        }
```

Comment:
```suggestion
        if (failureMsg != null) {
            throw new RuntimeException(failureMsg);
        }
```
`testStatus` is redudant.